### PR TITLE
Website: change <title> for SEO research

### PR DESCRIPTION
**Pull request by TuvalSimha**
Signed-off-by: Tuval Simha <tuval.simha@gmail.com>

### CODE REVIEW

**Review by B2o5T — CHANGES_REQUESTED** [review]
What the purpose? You will change the title of index page

File: website/theme.config.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export default defineConfig({
     return {
       description: description || `${siteName}: Modern API Platform and Ecosystem that scales`,
       openGraph: {
+        title: 'The Guild - GraphQL Tools',
```
Comment (B2o5T) [inline website/theme.config.tsx:36]: Why this change?

Comment (TuvalSimha) [reply website/theme.config.tsx:36]: Screenshot 2022-11-28 at 11 54 51
To change the <meta property="og:title" content="Open Source – The Guild">
@B2o5T

Comment (B2o5T) [inline website/theme.config.tsx:36]:
```suggestion
```

**Review by B2o5T — APPROVED** [review]
